source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -469,7 +469,7 @@ Use the Windows Registry Editor to navigate to the following key:
 
 HKCU\Software\Policies\Microsoft\Office\Common\Security
 
-If the value UFIControls exists, this is a finding.</RawString>
+If the value UFIControls exists, this is not a finding.</RawString>
```

Comment:
This rule needs to be reevaluated, the new guidance specifies "this is not a finding" whereas before it "this is a finding".  The correct way to perform this is to use the admx and configure the policy, then evaluate the registry for the correct setting.

Reply:
This rule appears to be correct. Configured via gpedit

![proplus472](https://user-images.githubusercontent.com/20539446/128450409-ad0e5220-eecf-4404-89fa-47d93ed218c4.JPG)

Reply:
Check again, our parsed rule has "Ensure = Absent", if it's present it isn't a finding...

Reply:
Should have ValueData defined now that it's "Present"